### PR TITLE
[diff.mods.to.definitions] Add entry for 'nullptr_t in stddef.h'

### PR DESCRIPTION
**Pull request by tkoeppe**

### CODE REVIEW

**Comment by tkoeppe** [comment]
Preview:

image

**Comment by tkoeppe** [comment]
Updated:

image

File: source/compatibility.tex [hunk 1]
```diff
@@ -2295,6 +2295,13 @@
 
 Subclause \ref{cwchar.syn} describes the changes.
 
+\pnum
+Headers \tcode{<cstddef>}~(\ref{cstddef.syn})\indexlibrary{\idxhdr{cstddef}}
```
Comment (zygoloid) [inline source/compatibility.tex]: I would suggest only mentioning `<cstddef>` here; the other clauses herein don't mention the corresponding `<stddef.h>`. If we want to draw attention to this, perhaps D.4 would be the place to do it.

Comment (tkoeppe) [inline source/compatibility.tex]: Good point, PTAL.

**Comment by sigfpe** [comment]
Agreed. I don't think that change is editorial.
-------- Original message --------From: Richard Smith <notifications@github.com> Date: 11/16/16  2:22 PM  (GMT-08:00) To: cplusplus/draft <draft@noreply.github.com> Subject: Re: [cplusplus/draft] [diff.mods.to.definitions] Add entry for
  'nullptr_t in stddef.h' (#1056) 
@zygoloid commented on this pull request.



In source/compatibility.tex:
> @@ -2295,6 +2295,13 @@
 
 Subclause \ref{cwchar.syn} describes the changes.
 
+\pnum
+Headers \tcode{<cstddef>}~(\ref{cstddef.syn})\indexlibrary{\idxhdr{cstddef}}

I would suggest only mentioning <cstddef> here; the other clauses herein don't mention the corresponding <stddef.h>. If we want to draw attention to this, perhaps D.4 would be the place to do it.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/cplusplus/draft","title":"cplusplus/draft","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/cplusplus/draft"}},"updates":{"snippets":[{"icon":"PERSON","message":"@zygoloid commented on #1056"}],"action":{"name":"View Pull Request","url":"https://github.com/cplusplus/draft/pull/1056#pullrequestreview-8922366"}}}

**Pull request merged**